### PR TITLE
Convert all UK spelling to US

### PR DESCRIPTION
**Pull request by joepetrowski**
Fixes https://github.com/paritytech/substrate/issues/1768

The only remaining thing to change is a crate in libp2p called `behaviour`.

### CODE REVIEW

**Comment by shawntabrizi** [comment]
Ran a script across all the files and found these too. You should be able to cherry pick commits if you don't like any of the changes I introduced

Still missing: `behaviour` to `behavior`

**Comment by shawntabrizi** [comment]
Oops, I forgot to deal with capitalization. Sorry, will fix.

**Comment by joepetrowski** [comment]
[Councilor is still used more in US English.](https://books.google.com/ngrams/graph?content=councilor%2Ccouncillor&case_insensitive=on&year_start=1800&year_end=2008&corpus=17&smoothing=3&share=&direct_url=t4%3B%2Ccouncilor%3B%2Cc0%3B%2Cs0%3B%3Bcouncilor%3B%2Cc0%3B%3BCouncilor%3B%2Cc0%3B%3BCOUNCILOR%3B%2Cc0%3B.t4%3B%2Ccouncillor%3B%2Cc0%3B%2Cs0%3B%3Bcouncillor%3B%2Cc0%3B%3BCouncillor%3B%2Cc0%3B%3BCOUNCILLOR%3B%2Cc0)

[Same with marshal](https://books.google.com/ngrams/graph?content=marshal%2C+marshall&case_insensitive=on&year_start=1800&year_end=2008&corpus=17&smoothing=3&share=&direct_url=t4%3B%2Cmarshal%3B%2Cc0%3B%2Cs0%3B%3BMarshal%3B%2Cc0%3B%3Bmarshal%3B%2Cc0%3B%3BMARSHAL%3B%2Cc0%3B.t4%3B%2Cmarshall%3B%2Cc0%3B%2Cs0%3B%3BMarshall%3B%2Cc0%3B%3BMARSHALL%3B%2Cc0)

[cancelled/canceled is basically a tie](https://books.google.com/ngrams/graph?content=canceled%2C+cancelled&case_insensitive=on&year_start=1800&year_end=2008&corpus=17&smoothing=3&share=&direct_url=t4%3B%2Ccanceled%3B%2Cc0%3B%2Cs0%3B%3Bcanceled%3B%2Cc0%3B%3BCanceled%3B%2Cc0%3B.t4%3B%2Ccancelled%3B%2Cc0%3B%2Cs0%3B%3Bcancelled%3B%2Cc0%3B%3BCancelled%3B%2Cc0%3B%3BCANCELLED%3B%2Cc0)

That's why I didn't do those three. Good catches on the others.

**Comment by joepetrowski** [comment]
My search for colour comes up empty, where do you see it? Behaviour I mentioned in my comment, will need to update libp2p.

```
parity@eta ~/p/s/substrate> grep -rnIe 'colour'
parity@eta ~/p/s/substrate> 
```

**Comment by gavofyork** [comment]
Indeed. Some of those changes will need rolling back.

We use:
- Councillor (https://english.stackexchange.com/questions/350196/councilor-vs-councillor)
- Cancellation ("cancelation" is plain wrong, even in the US: https://www.grammarly.com/blog/canceled-vs-cancelled/)

I would basically go with the most common spelling on google for everything. "Google English".

**Comment by gavofyork** [comment]
Otherwise, it looks good.

File: core/rpc/src/chain/mod.rs [hunk 1]
```diff
@@ -57,9 +57,9 @@ pub trait ChainApi<Number, Hash, Header, SignedBlock> {
 	#[rpc(name = "chain_getBlockHash", alias("chain_getHead"))]
 	fn block_hash(&self, hash: Option<number::NumberOrHex<Number>>) -> Result<Option<Hash>>;
 
-	/// Get hash of the last finalised block in the canon chain.
-	#[rpc(name = "chain_getFinalisedHead")]
-	fn finalised_head(&self) -> Result<Hash>;
+	/// Get hash of the last finalized block in the canon chain.
+	#[rpc(name = "chain_getFinalizedHead")]
```
Comment (jacogr) [inline core/rpc/src/chain/mod.rs]: Can we create an alias for the old names? From a middleware perspective there is no way to support both types of chains. So really from a use perspective in the API and UIs this rename is quite problematic without an alias for the old - there needs to be time to swap over.

**Comment by shawntabrizi** [comment]
@joepetrowski you can find `Colour` in two files:
https://github.com/paritytech/substrate/blob/master/core/cli/src/informant.rs#L19
https://github.com/paritytech/substrate/blob/master/core/cli/src/lib.rs#L714

Again, these are part of an external crate, so not much we can do, but just an FYI

File: core/rpc/src/chain/mod.rs (hunk 1 above)
Comment (gavofyork) [reply core/rpc/src/chain/mod.rs]: Not sure if aliases are possible. If not I'd suggest we just bite the bullet and only support post 1.0 from here on.

Comment (tomusdrw) [reply core/rpc/src/chain/mod.rs]: aliases are supported via `#[rpc(name = ".", alias("a"), alias("b")]`

File: core/rpc/src/chain/mod.rs [hunk 2]
```diff
@@ -81,19 +81,19 @@ pub trait ChainApi<Number, Hash, Header, SignedBlock> {
 
 	/// New head subscription
 	#[pubsub(
-		subscription = "chain_finalisedHead",
+		subscription = "chain_finalizedHead",
```
Comment (tomusdrw) [inline core/rpc/src/chain/mod.rs]: However, we don't support aliases for subscription names, so this has to use the new name despite being subscribed via alias.

**Pull request merged**